src/module_loader.rs

```diff
@@ -247,7 +324,7 @@ mod test {
             .get(&specifier)
             .expect("Expected to get cached source");
 
-        let loader = RustyLoader::new(Some(Box::new(cache_provider)));
+        let loader = RustyLoader::new(Some(Box::new(cache_provider)), None);

```

Comment:
Could you add a new test verifying the trait?

Reply:
Will do.